src/containers/forgotPassword/index.tsx

```diff
@@ -33,19 +40,39 @@ const ForgotPassword: React.FC = () => {
       </Helmet>
       <ContentContainer>
         <PageHeader
-          pageTitle="Forgot Password"
+          pageTitle="Forgot your password?"
           isMobile={isMobile(windowType)}
         />
-        <Form name="forgotPassword" onFinish={onFinish}>
-          <Form.Item name="email" rules={enterEmailRules}>
-            <Input placeholder="Email" />
-          </Form.Item>
-          <Form.Item>
-            <Button type="primary" htmlType="submit" size="large">
-              Submit
-            </Button>
-          </Form.Item>
-        </Form>
+        {!submittedEmail || userNotFound ? (
+          <>
+            <Typography.Title level={4}>
+              Please enter your email and we will send you the password reset
+              instructions to the email address for this account.
+            </Typography.Title>
+            <Form form={form} name="forgotPassword" onFinish={onFinish}>
+              {userNotFound && (
+                <Typography.Title level={4}>
+                  We couldn't find an account associated with the given email.
```

Comment:
I don't feel too good showing this message to the users because it technically allows attackers to potentially submit a bunch of different emails to see which ones are in use but I'm also not sure how common this would even be and/or how valuable information gained from this message would be to attackers.

Reply:
I agree, this is a big yikes from a security point of view.

Reply:
Good point, you can switch this to a message that just says "If an account is registered under [given email], we have sent you a link to reset your password."

🔧 Could you also change the `LOGIN_ERROR` message under assets/content.tsx to 'The email or password you entered was incorrect'? Not in the scope of this ticket, but it should be a quick fix and is relevant to this idea of hiding what emails are registered.